### PR TITLE
Graph Davidppp0.png

### PR DESCRIPTION
**Pull request by Davidppp0**
Draft of the graph.
If you see an error, please let me know.

### CODE REVIEW

**Comment by Davidppp0** [comment]
Add ttl

File: Assignment2/Davidppp0.ttl [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+@prefix objeto: <http://oeg.fi.upm.es/resource> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+ 
+objeto:ClassRoom03  tortuga:contains objeto:Sensor347 ;
+              tortuga:contains objeto:Table322 . 
+
+objeto:Sensor347  tortuga:hasMeasurement objeto:Measuremment8432 .
+
+objeto:Measuremment8432  tortuga:hasTemperature "29"^^xsd:double ;
+                   tortuga:atTime "2022-09-12T12:01:12"^^xsd:date .
```
Comment (ocorcho) [inline Assignment2/Davidppp0.ttl]: xsd:dateTime

File: Assignment2/Davidppp0-fromJSONtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+    "@context": {
+        "sch":"http://schema.org/",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "id": "rdfs:label",
+        "type": "rdfs:label",
+        "fullName": "rdfs:label",
+        "hometown": "rdfs:label",
+        "age": "rdfs:label",
```
Comment (ocorcho) [inline Assignment2/Davidppp0-fromJSONtoJSON-LD.jsonld:9]: Modify "rdfs:label" into schema:age or alike

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Comment by ocorcho** [comment]
I still see the issue with the JSON-LD, but accepted.

**Pull request merged**